etc/evince.profile

```diff
@@ -38,7 +38,6 @@ private-etc fonts
 
 #private-lib - seems to be breaking on Gnome Shell 3.26.2, Mutter WM, issue 1711
 #private-lib evince,libpoppler-glib.so.8
```

Comment:
You can delete this line since it's being replaced by the new private-lib filter. ;)